test/self_driving/modeling_test.cpp

```diff
@@ -38,14 +39,23 @@ class ModelServerTest : public TerrierTest {
 
     return db_main;
   }
+
+  /**
+   * Wrapper to hide long chain of function name resolution
+   * @param unit_type OpUnit
+   * @return  string representation of the opunit
+   */
+  static const std::string opunit_string(selfdriving::ExecutionOperatingUnitType unit_type) {
```

Comment:
nit: based on our naming convention, this should be `OpUnitToString`.

Reply:
Err right, what was I thinking...too much python/c